Makefile

```diff
@@ -36,14 +36,17 @@ initContainer: fmt vet
 docker-publish-initContainer: docker-build-initContainer docker-tag-repo-initContainer docker-push-initContainer
```

Comment:
Since `docker-push-initContainer` does everything `build` `tag` `push`, can we remove `docker-publish-initContainer`?